code/modules/experisci/experiment/handlers/experiment_handler.dm

```diff
@@ -72,7 +72,7 @@
 	// Note this won't work at the moment for non-machines that have been included
 	// on the map as the servers aren't initialized when the non-machines are initializing
 	if (!(config_flags & EXPERIMENT_CONFIG_NO_AUTOCONNECT))
-		var/list/found_servers = get_available_servers(parent)
+		var/list/found_servers = get_available_servers()
 		var/obj/machinery/rnd/server/selected_server = length(found_servers) ? found_servers[1] : null
 		if (selected_server)
 			link_techweb(selected_server.stored_research)
```

Comment:
doesn't this mean we can link to a techweb we like, aren't on the same z level as? what does this do to uh, golems?

Reply:
It means you need at least 1 RD server to connect to the techweb, this doesn't effect things like RD consoles as of yet, so this means Golems can't do experiments without building a server first.
My idea is in the future off-station roles will get an RD server that gives its own techweb, so Golems/Oldstation will have their own research to do instead of taking the station's. I have to write up a hackmd for it to flesh it all out though.